hack/aks/Makefile

```diff
@@ -90,10 +90,12 @@ overlay-byocni-up: rg-up overlay-net-up ## Brings up an Overlay BYO CNI cluster
 	$(AZCLI) aks create -n $(CLUSTER) -g $(GROUP) -l $(REGION) \
 		--node-count $(NODE_COUNT) \
 		--node-vm-size $(VM_SIZE) \
-		--load-balancer-sku basic \
+		--load-balancer-sku standard \
 		--network-plugin none \
 		--network-plugin-mode overlay \
-		--pod-cidr 192.168.0.0/16 \
```

Comment:
a bunch of the overlay and cilium tests implicitly depend on this value being 192.168.0.0/16, why do we need to change it?

Reply:
emphasis on the `do-not-merge` tag, I'm using this target for a convenient one-click byocni cilium overlay cluster for manual testing

I believe I can still use `refs/pull/2055/merge` as a target if pr is converted to draft